templates/web/legacy/embedded_legacy_form.ftl

```diff
@@ -158,6 +158,9 @@
                                     });
                                   }
                                 },
+                                pendingChanges: () => {
+                                  window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_PENDING_CHANGES', action: 'pendingChanges'}, '*');
```

Comment:
Did you mean `payload`? What's `action`?
```suggestion
                                  window.top.postMessage({ type: 'EMBEDDED_LEGACY_FORM_PENDING_CHANGES', payload: 'pendingChanges'}, '*');
```

Reply:
this is used as an ID for second type(), for example when the type called is `EMBEDDED_LEGACY_FORM_SAVE`, there are 3 second types:  'save',  'saveAndClose',  'saveAndPreview', and that is the value of action, for the other actions is not needed to send a type, but can be useful

Reply:
`                      window.top.postMessage({
                        ...response,
                        type: 'EMBEDDED_LEGACY_FORM_SAVE',
                        refresh: false,
                        tab: type,
                        redirectUrl: response.item?.browserUri,
                        action,
                        isNew: true
                      }, '*');`

Reply:
these changes include moving all the parameters to payload, should I proceed?